web/common.go

```diff
@@ -0,0 +1,15 @@
+package web
+
+import (
+	"context"
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Since some URLs are triggered by App Engine cron, error messages returned in HTTP response
+// might not be visible to humans. We need to log them as well, and this helper function does that.
+func logAndReturnError(ctx context.Context, w http.ResponseWriter, err error) {
```

Comment:
nit: Do you really need a separate file for this function?

Reply:
Now that I pulled the other function out, no. Moved to handlers.